mingw32.sh

```diff
@@ -14,9 +14,7 @@ unset PKG_CONFIG_PATH
 # Removed -static from the following
 export CFLAGS="-static-libgcc"
 export CXXFLAGS="-static-libgcc -static-libstdc++"
-export CPPFLAGS="-I$HOME/.local/$target/include -O2"
-export LDFLAGS=-L$HOME/.local/$target/lib
-export PKG_CONFIG_LIBDIR=$HOME/.local/$target/lib/pkgconfig
-export PATH=$HOME/.local/$target/bin:$PATH
+export CPPFLAGS="-O2"
+export PKG_CONFIG_LIBDIR=/usr/i686-w64-mingw32/sys-root/mingw/lib/pkgconfig/
```

Comment:
Only available on Fedora, can you somehow generalize it to cover the way we used it

Reply:
I just checked, PKG_CONFIG_LIBDIR can be a list separated by colon also; so the originals of these 4 are harmless on Fedora if .local/target is not there.